src/it/java/com/zotov/edu/passportofficerestservice/PersonBaseTest.java

```diff
@@ -0,0 +1,136 @@
+package com.zotov.edu.passportofficerestservice;
+
+import com.zotov.edu.passportofficerestservice.model.ErrorMessage;
+import com.zotov.edu.passportofficerestservice.model.Person;
+import com.zotov.edu.passportofficerestservice.model.PersonsPageResponse;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static io.restassured.RestAssured.given;
+import static io.restassured.RestAssured.when;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public abstract class PersonBaseTest extends BaseTest {
+
+    void verifyPersonsPageResponse(List<Person> expectedPersons, PersonsPageResponse personsPageResponse,
+                                   int pageSize, int pageNumber) {
+        int totalElements = expectedPersons.size();
+        int totalPages = totalElements == 0 ? 0 : totalElements <= pageSize ? 1 : totalElements / pageSize;
+        List<Person> expectedPersonsPage = expectedPersons
```

Comment:
can it be more explicit in tests? So that it's not calculated but explicitly defined in test that calls this `verify..` method

Reply:
Probably, we can create expected `PersonsPageResponse` and verify that actual response is equal to an expected one.

Reply:
replaced by the verifying of PageResponse